src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderTests.cs

```diff
@@ -304,5 +304,67 @@ public void Dispose()
                 }
             }
         }
+
+        [Fact]
+        public static void VerifyHashSignature_SHA256_2048()
```

Comment:
>Since the other methods are covered in RSACryptoServiceProviderBackCompat.cs, please add this there.

Done.

>Something like new SignHash to old VerifyHash, and vice versa. Doesn't need to be the big known value test case, as long as we test in terms of something else already being tested.

I renamed it to OldVerifyHash and put in an effort to add the "vice versa" case and remove the big known value. This is the first time I've ever touched crypto so I may have misinterpreted what you meant though.